src/App.js

```diff
@@ -1,26 +1,28 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
+import { Route, Switch} from 'react-router-dom'
+import SearchPage from './SearchPage.js'
+import Header from './Header.js'
+import PokemonDetail from './PokemonDetail.js'
+import './App.css'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export default class App extends Component {
+    render() {
+        return (
+            <div className='container'>
+                <Header />
+                <div className ='searchBox'>
+                <Switch>
+                    <Route
+                    path="/home"
```

Comment:
If you set this path to '/' instead of '/home' it will load the homepage on start.